math/genvector/inc/Math/GenVector/DisplacementVector3D.h

```diff
@@ -249,11 +249,13 @@ namespace ROOT {
        */
       template <class IT>
       void GetCoordinates( IT begin) const {
-         Scalar a,b,c = 0;
-         GetCoordinates (a,b,c);
-         *begin++ = a;
-         *begin++ = b;
-         *begin = c;
+        Scalar a = Scalar(0);
+        Scalar b = Scalar(0);
+        Scalar c = Scalar(0);
```

Comment:
My way avoids a potential default construction followed by assignment, so could be more efficient.